Application/src/visuals/Draw.java

```diff
@@ -164,11 +174,12 @@ public void update() {
      * Renders the message when trying to resize the application.
      */
     void renderMessage() {
+        if(messages.size() == 0) return; // There are no messages in the list.
         if(this.messages.size() > 0) {
-            for (int i = 0; i < messages.size(); i++) {
+            for (int i = 0; i < messages.size(); i++) { //Messages exist and will now be be re-placed.
```

Comment:
Use a iterator instead of a for i loop.

Reply:
Not part of this pull-request but i can fix it